go.mod

```diff
@@ -95,7 +90,7 @@ require (
 	github.com/BurntSushi/toml v1.3.2
 	github.com/Jeffail/gabs v1.4.0
 	github.com/amazon-contributing/opentelemetry-collector-contrib/extension/awsmiddleware v0.0.0-20240709194807-b0f0c0eda01b
```

Comment:
Change this one and `github.com/amazon-contributing/opentelemetry-collector-contrib/override/aws` in the indirect require block as well to match the other contrib versions.